atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockTables.java

```diff
@@ -53,13 +66,34 @@ public SchemaMutationLockTables(CassandraClientPool clientPool, CassandraKeyValu
                 .collect(Collectors.toSet());
     }
 
+    public boolean isLockTableEmpty(TableReference tableRef) {
+        ColumnPath columnPath = new ColumnPath(tableRef.getQualifiedName());
+        columnPath.setColumn(SchemaMutationLock.getGlobalDdlLockColumnName());
+        try {
+            clientPool.runWithRetry(client -> client.get(
+                    SchemaMutationLock.getGlobalDdlLockRowName(), columnPath, ConsistencyLevel.LOCAL_QUORUM));
+        } catch (NotFoundException e) {
+            return true;
+        } catch (TException e) {
+            return false;
+        }
+
+        return false;
```

Comment:
Should be moved into the try block, perhaps with a comment that successfully running the query means the table was non-empty, and we don't care about the contents.